packages/jitm/.gitignore

```diff
@@ -0,0 +1,2 @@
+## Ignore composer.lock for our library packages
+/**/composer.lock
```

Comment:
That one shouldn't be necessary, as there is now one in `/packages` that should be taking care of this one.